### PR TITLE
Fix play console bugs

### PR DESCRIPTION
**Pull request by deeppandya**
Resolves https://github.com/brave/brave-browser/issues/14107

## Submitter Checklist:

- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [x] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [x] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [x] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [x] Ran `git rebase master` (if needed)
- [ ] Requested a security/privacy review as needed

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

### CODE REVIEW

File: android/java/org/chromium/chrome/browser/onboarding/OnboardingPrefManager.java [hunk 1]
```diff
@@ -232,7 +232,7 @@ public void showOnboarding(Context context) {
 
     public void onboardingNotification(Context context) {
```
Comment (SergeyZhukovsky) [inline android/java/org/chromium/chrome/browser/onboarding/OnboardingPrefManager.java:233]: do we need the `context` argument here?

File: android/java/org/chromium/chrome/browser/onboarding/P3aOnboardingActivity.java [hunk 2]
```diff
@@ -46,13 +47,23 @@ protected void onCreate(Bundle savedInstanceState) {
                         ? getResources().getString(R.string.p3a_onboarding_title_text_1)
                         : getResources().getString(R.string.p3a_onboarding_title_text_2));
         CheckBox p3aOnboardingCheckbox = findViewById(R.id.p3a_onboarding_checkbox);
-        p3aOnboardingCheckbox.setChecked(BravePrefServiceBridge.getInstance().getP3AEnabled());
+        boolean isP3aEnabled = true;
+        try {
+            isP3aEnabled = BravePrefServiceBridge.getInstance().getP3AEnabled();
+        } catch (Exception e) {
+            Log.e("P3aOnboarding", e.getMessage());
```
Comment (SergeyZhukovsky) [inline android/java/org/chromium/chrome/browser/onboarding/P3aOnboardingActivity.java:54]: let's do `isP3aEnabled = false` here, so it's disabled if failed

Comment (deeppandya) [reply android/java/org/chromium/chrome/browser/onboarding/P3aOnboardingActivity.java:54]: @SergeyZhukovsky by default it should be enabled according to spec.

Comment (deeppandya) [reply android/java/org/chromium/chrome/browser/onboarding/P3aOnboardingActivity.java:54]: User can still uncheck the selection from UI

**Review by SergeyZhukovsky — APPROVED** [review]
++

**Pull request merged**